bin/merge_sces.R

```diff
@@ -109,6 +109,18 @@ if (!all(sce_checks)) {
   )
 }
 
+sce_list <- sce_list |>
+  purrr::map(\(sce){
+    # add a new column with any additional modalities
+    # will be adt, cellhash, or NA
+    additional_modalities <- altExpNames(sce)
+    if (length(additional_modalities) == 0) {
+      additional_modalities <- NA
+    }
+    sce$additional_modalities <- additional_modalities
```

Comment:
Is this simpler? Also, maybe this should be in `metadata` instead of `colData`? Use what you want, don't use what you don't want!

```suggestion
    metadata(sce)$additional_modalities <- ifelse( 
      length(additional_modalities) == 0, 
      NA, 
      additional_modalities
    )
```

Reply:
I actually want this in `colData` so that when I merge objects I have a final column in the merged SCE object that tells me the modalities for each library. This accounts for scenarios where projects have some but not all libraries with ADT or with multiplexing.